assignments/prototypes.js

```diff
@@ -32,6 +53,23 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+
+function Humanoid(human) {
+  GameObject.call(this, Humanoid);
+  this.team = human.team;
+  this.weapons = human.weapons;
+  this.language = human.language;
+ 
+  CharacterStats.call(this, human);
+}
+
+Humanoid.prototype.destroy = Object.create(CharacterStats.prototype);
+Humanoid.prototype.takeDamage = Object.create(CharacterStats.prototype);
```

Comment:
same issue here.  It's more obvious here, as you can see the `destroy` and `takeDamage` properties aren't purple to indicate that they're methods.